docs/docs/concepts.mdx

```diff
@@ -57,6 +60,12 @@ A developer platform that lets you debug, test, evaluate, and monitor LLM applic
   title="LangChain Framework Overview"
 />
 
+- **[LangGraph Cloud](https://langchain-ai.github.io/langgraph/cloud/)**:
```

Comment:
I'd be consistent with the website language here: 
"LangGraph Cloud: The infrastructure that lets you run agents at scale. Get 1-click deployment, scalable servers and task queues, and integrated monitoring to streamline agentic workflows.

Check out the LangGraph Cloud docs: https://langchain-ai.github.io/langgraph/cloud/"